src/utils/toastUtils.ts

```diff
@@ -0,0 +1,42 @@
+import { toast } from "react-toastify";
+
+export const ToastHandler = (type: string, message: string) => {
+  switch (type) {
+    case "success": {
+      toast.success(message, {
+        position: "top-center",
+        autoClose: 3000,
+        draggable: true,
+      });
+
+      break;
+    }
+    case "error": {
+      toast.error(message, {
+        position: "top-center",
+        autoClose: 3000,
+        draggable: true,
+      });
+
+      break;
+    }
+    case "warn": {
+      toast.warn(message, {
+        position: "top-center",
+        autoClose: 3000,
+        draggable: true,
+      });
```

Comment:
This config is repetitive, can we create one config constant and use it everywhere?
```js
{
        position: "top-center",
        autoClose: 3000,
        draggable: true,
      }
```